dipy/viz/ui.py

```diff
@@ -1939,6 +1939,7 @@ def __init__(self, position=(0, 0),
 
         # By setting the value, it also updates everything.
         self.value = initial_value
+        self.pvalue = initial_value
```

Comment:
This too needs to be added to the docstring. 
Also, IMO an explicit `previous_value` would be a better name for this attribute.